src/NewsExtraFieldDisplay.php

```diff
@@ -135,6 +135,7 @@ public function formAlter(array &$form, FormStateInterface $form_state, $form_id
       ($form_display->getComponent('localgov_news_newsroom_promote'))
     ) {
 
+      /** @var \Drupal\node\NodeForm $form_object */
```

Comment:
I'd rather use `assert`ions now that we can, rather than IDE magics.

Reply:
Hi @ekes I've refactored this so the correct types are used and added a check to make sure `$form_object` is `NodeForm`